service/README.rst

```diff
@@ -190,25 +189,6 @@ restored to the value they had when the session was converted,
 regardless of whether or not they were adjusted during the session
 through the service.
 
-The request to open a session is done in the ServiceIOGroup
-constructor, and the request to close the session is made by the
-ServiceIOGroup destructor.  Calls to the ServiceIOGroup's
-``write_control()`` or ``push_control()`` methods will trigger the
-conversion of the session to write mode.  Calls to these methods will
-only occur when the ServiceIOGroup is the only loaded IOGroup that
-provides the control requested by the user since all IOGroups are
-loaded by the PlatformIO factory after the ServiceIOGroup.
-
-Note that if any control adjustments are made during a session through
-the GEOPM Service then every control supported by GEOPM will be
-reverted when the session ends.  One consequence of this is that when
-a control is exposed to a user only through the GEOPM Service, then
-the geopmwrite command line tool will not be effective (the value will
-be written, but reverted when the geopmwrite process ends).  The
-geopmsession command line tool can be used to write any number of the
-GEOPM supported controls and keep a session open for a specified
-duration (or until the geopmsession process is killed).
```

Comment:
Just to understand: is the reason for removing these paragraphs that this level of detail perhaps belongs in the developer's doc/guide?

Reply:
The first paragraph was deleted because it is really an implementation detail of the ServiceIOGroup, and way to specific for the service top level readme.  The second paragraph was deleted because it is no longer true.  The write lock is tied to the session leader process, which is not the geopmwrite process in almost all use cases.  The geopmsession tool also no longer supports write mode exactly for this reason.